R/utils.R

```diff
@@ -398,3 +398,8 @@ is_keras_tensor <- function(x) {
     k_is_tensor(x)
   }
 }
+
+
+# conflict with utils::zip, maybe another name?
+zip <- function(..., simplify = TRUE)
```

Comment:
does `py_zip` make sense ...? or even .... move this to `reticulate`?

Reply:
I would expect `py_zip` to be a wrapper around `reticulate::import_builtins()$zip`.